File: app/src/main/java/ru/SnowVolf/convertx/regexdragon/SpurFragment.java
package ru.SnowVolf.convertx.regexdragon;

import android.graphics.Typeface;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v7.view.menu.MenuBuilder;
import android.view.LayoutInflater;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

import ru.SnowVolf.convertx.R;

/**
 * Created by Snow Volf on 02.03.2017, 20:31
 */

public class SpurFragment extends Fragment{
    private TextView spurr;
    private View rootView;

    @Override
    public void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);
        setRetainInstance(true);
        setHasOptionsMenu(true);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState){
        rootView = inflater.inflate(R.layout.fragment_spur, container, false);
        spurr = (TextView) rootView.findViewById(R.id.regex_spur);
        return rootView;
    }

    @Override
    public void onDestroyView(){
        super.onDestroyView();
        rootView = null;
    }

    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
        if (menu != null) {
            menu.clear();
        } else
            menu = new MenuBuilder(getContext());
        menu.add("").setShowAsActionFlags(MenuItem.SHOW_AS_ACTION_ALWAYS);
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        if (item.getItemId() == android.R.id.home)
            getActivity().finish();
        return true;
    }

    public void onActivityCreated(Bundle savedInstanceState){
        super.onActivityCreated(savedInstanceState);

        final StringBuilder sb = new StringBuilder();
        try {
            BufferedReader br = new BufferedReader(new InputStreamReader(getContext().getAssets().open("SPUR.md"), "UTF-8"));
            String line;
            while ((line = br.readLine()) != null) {
                sb.append(line).append("\n");
            }
            br.close();
        } catch (IOException e) {
            e.printStackTrace();
        }
        Typeface Mono = Typeface.createFromAsset(getContext().getAssets(), "fonts/RobotoMono-Regular.ttf");
        spurr.setTypeface(Mono);
        spurr.setText(sb);

    }
}

File: app/src/main/java/ru/svolf/convertx/presentation/screens/regex/SpurFragment.kt
package ru.svolf.convertx.presentation.screens.regex

import android.content.Context
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import ru.svolf.convertx.databinding.FragmentSpurBinding

/**
 * Created by Snow Volf on 02.03.2017, 20:31
 */
class SpurFragment : Fragment() {
    private var _binding: FragmentSpurBinding? = null
    private val binding get() = _binding!!
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
		_binding = FragmentSpurBinding.inflate(inflater, container, false)
		return binding.root
	}

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        binding.regexSpur.text = readTextFileFromAssets(requireContext(), "SPUR.md")
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }

    private fun readTextFileFromAssets(context: Context, fileName: String): String {
        val inputStream = context.assets.open(fileName)
        val buffer = ByteArray(inputStream.available())
        inputStream.read(buffer)
        inputStream.close()
        return String(buffer)
    }
}